docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -60,11 +60,17 @@ alluxio.master.mount.table.root.ufs=<STORAGE_URI>
   mount to the Alluxio root.
   This shared storage system must be accessible by the master node and all worker nodes.
   Examples include `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`, or 
-  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`.
+  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`
+  (See [this tutorial](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html#running-alluxio-locally-with-s3) 
+  for more details about configuring Amazon S3 as Alluxio's under storage system).
+
+Then, append the host name of all the Alluxio master nodes into `conf/masters` 
+and the host name of all the Alluxio workers node into `conf/workers`.
 
 Next, copy the configuration file to all the Alluxio worker nodes.
 The following built-in utility will copy the configuration files to all master and worker
 nodes specified in the `conf/masters` and `conf/workers` files respectively.
+Note that the inbound rules and ports of each node need to allow access from all the other nodes in the cluster.
```

Comment:
this point was covered in the above prerequisites section:
```
* TCP traffic across all nodes is allowed.
  For basic functionality, make sure RPC port (default:19998) is open on all nodes.
```
this statement is also overly broad; normally you would not want to allow all ports of one node to be accessible to every other node in the cluster. this could cause further misunderstandings about the requirements needed to deploy alluxio in a cluster. i would not include this note here.

if you think there needs to be more detailed explanation about this topic, it would fit better alongside the prerequisites section